core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/IoUtil.java

```diff
@@ -134,6 +141,51 @@ public static InputStream getResource(String resource) throws IOException {
         return is;
     }
 
+    /**
+     * Provides streams to all resources matching {@code filter} criteria.
+     * Method search inside archive if context represents a jar file.
+     *
+     * @param context
+     *      the current context
+     * @throws IOException
+     */
+    public static void getResources(Class context, Pattern filter, Consumer<InputStream> consumer) throws IOException {
```

Comment:
This is very custom and very specific logic. Used only once. I recommend to move it to place there it used.
BTW do you have a test for it?